app/README.md

```diff
@@ -96,6 +123,10 @@ npm run create-admin
 npm run dev
 ```
 
+The standard port is 3000 and the application can be opend at http://localhost:3000. Please check the log outputs, if a different port has been used.
+
+Use `johndeo@example.com` and `password` to login.
```

Comment:
I don't think we create this example account? You need to sign up first.

Reply:
Also it's johndoe

Reply:
In the example.env file it is listed as:

DEFAULT_ADMIN_EMAIL=johndeo@example.com
DEFAULT_ADMIN_PASSWORD=password

For me it worked like this

Reply:
That is happening in the script 

```
## set up login in credentials
npm run create-admin
```